roles/ceph-osd/defaults/main.yml

```diff
@@ -4,15 +4,15 @@ ceph_osd:
   pkgs:
     - parted
 
-  # ONLY SET THIS TO TRUE IF YOU KNOW WHAT YOU'RE DOING.
-  # This will likely only be used on a fresh install.
-  destroy_devices_no_matter_what: false
+  # choose one scenario
+  journal_collocation: false

```

Comment:
should this just be a true/false statement with the false statement meaning that `raw_multi_journal` is used ?   otherwise a person could set both to true and cause weirdness ?